index.html

```diff
@@ -51,19 +67,20 @@ <h2>Pick Your Destination</h2>
       <div class="destination">
         <h4>Fun In The Sun</h4>
         <p>Expedition excursion design excursion fun, clean simple organized WordPress Travel colorful webdesign. Traveler blogger website design expedition clean excursion traveling.</p>
-        <div class="btn">Sign Me Up!</div>
+        <div><button class="btn">Sign Me Up!</div>
       </div>
```

Comment:
`button` tags should have an opening and closing tag. So you could keep it a `div` with a class of `btn` or change it to `<button></button>`